cider-velox/src/CiderStatefulOperator.cpp

```diff
@@ -43,6 +45,13 @@ RowVectorPtr CiderStatefulOperator::getOutput() {
   if (ret == CiderRuntimeModule::ReturnCode::kNoMoreOutput) {
     finished_ = true;
   }
+  if (is_using_arrow_fromat_) {
+    auto [schema, array] = output_->move();
+    VectorPtr baseVec = importFromArrowAsOwner(*schema, *array, operatorCtx_->pool());
+    CiderBatchUtils::freeArrowArray(array);
+    CiderBatchUtils::freeArrowSchema(schema);
+    return std::reinterpret_pointer_cast<RowVector>(baseVec);
```

Comment:
move() allocate new `schema`, `array`, and then `freed` immediately, not perfect.

Reply:
add a new move method